show-land/sass/_top-nav.scss

```diff
@@ -0,0 +1,30 @@
+
+.main-nav{
+    &__list{
+display: flex;
+padding-top: 1rem;
+gap: 1.5rem;
+list-style: none;
+align-items: center;
+    }
+&__link:link,
+&__link:visited{
+    text-decoration: none;
+    font-size: 1rem;
+    color: $colorWihte;
+    text-transform: uppercase;
+    font-weight: 700;
+ 
+}
+
+}
+.header{
+    display: flex;
+    justify-content: space-between;
+  
+}
+.logo{
+    padding-top: 0.8rem;
+    font-size: 2rem;
+    font-weight: 700;
+}
```

Comment:
You can use one of the functions of sass, which is code nesting, I think it can help your code a lot and can make your code more beautiful

```
.container {
  .header {
    display: flex;
    justify-content: space-between;
  }

  .logo {
    padding-top: 0.8rem;
    font-size: 2rem;
    font-weight: 700;
  }

  .main-nav {
    &__list {
      display: flex;
      padding-top: 1rem;
      gap: 1.5rem;
      list-style: none;
      align-items: center;
    }
    &__link:link,
    &__link:visited {
      text-decoration: none;
      font-size: 1rem;
      color: $colorWihte;
      text-transform: uppercase;
      font-weight: 700;
    }
  }
}
```